src/components/DropdownForm/DropdownForm.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {
+  useCallback, useEffect, useRef, useState,
+} from 'react';
+import debounce from 'lodash.debounce';
+import { Person } from '../../types/Person';
+
+type Props = {
+  querry: string;
```

Comment:
The prop 'querry' is misspelled. It should be 'query'. This error is repeated in several parts of your code.